examples/navigation/firstperson/index.html

```diff
@@ -165,7 +165,7 @@
 		player.position.set( - 13, - 0.75, - 9 );
 
 		controls = new FirstPersonControls( player );
-		controls.lookSpeed = 2;
+		controls.lookingSpeed = 2;
```

Comment:
Do you mind deleting the lines which were `controls.lookSpeed = 2;`? I'd like to keep the current behavior and with setting `lookingSpeed` to `2` the controls are too sensitive.

Reply:
You misunderstand, the value was from you.
Either we remove this line (as `lookSpeed` is no where used), or we have to rename the variable in the controls (as it is named `lookingSpeed` there).

The problem is, you are writing to a property that is never read.
See the code in `controls.js`: https://github.com/Mugen87/yuka/blob/73c0581597954712e487eec52aea62a271685375/examples/navigation/firstperson/src/FirstPersonControls.js#L27
https://github.com/Mugen87/yuka/blob/73c0581597954712e487eec52aea62a271685375/examples/navigation/firstperson/src/FirstPersonControls.js#L152-L153

Reply:
Sorry, for not being clear 😅 . I've mean this:

> Either we remove this line (as lookSpeed is no where used),

Reply:
Never mind! I clean things up 👍 .

Reply:
Ah, ok, sorry. That got lost in translation. :smile_cat: 
All good then.